JavaScriptSPA/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+    height: 100%;
+    background-color: #2b5797; /* For browsers that do not support gradients */
+    background-image: linear-gradient(-90deg,  #2b5797, #603cba); /* Standard syntax (must be last) */
+}
+.container {
+    width: 100%;
+    min-height: 600px;
+}
+
+.leftContainer {
+    color: white;
+    font-size: 50px;

```

Comment:
Can we reduce the font size of the welcome message? When the username is long it is bleeding into the text box to the right.